fs_brokers/apache_hdfs_broker/src/main/java/org/apache/doris/broker/hdfs/FileSystemManager.java

```diff
@@ -586,6 +586,8 @@ public void closeReader(TBrokerFD fd) {
                 fsDataInputStream.close();
             } catch (IOException e) {
                 logger.error("errors while close file input stream", e);
+                throw new BrokerException(TBrokerOperationStatusCode.TARGET_STORAGE_SERVICE_ERROR,
+                        e, "errors while close file input stream");
```

Comment:
The md5sum was generated before the local file upload to remote, which means the local file is a empty file.
Is it normal we have a empty local file?

Reply:
Not because an empty file was uploaded, but because the uploaded file was not closed properly, resulting in an empty file.